services/monitor/monitor.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+#
+# SneezyMUD Container Monitor - Automatic image updates on container exit
+#
+set -e
```

Comment:
Also consider `set -eu -pipefail`.
The `u` flag makes the script barf when using unassigned variables.